solutions/usaco-815.mdx

```diff
@@ -0,0 +1,118 @@
+---
+id: usaco-815
+source: USACO Gold 2018 Febuary
+title: Taming the Herd
+author: Shijie Ren
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_gold_feb18.html)
+
```

Comment:
could you make a note here saying that your solution is O(N^2) (and that it's faster than the editorial's but not required since `N<=100`)?